README.md

```diff
@@ -606,6 +607,60 @@ $ remme transaction get \
 }
 ```
 
+### Receipt
+
+Get list of the transaction receipts by identifiers — ``remme receipt get``:
```

Comment:
`Get a list...`, `of the transaction's`. Also, apply to the documentation below and code documentation.

Reply:
@dmytrostriletskyi, done.